### DISCUSSION TITLE
Masalah rubygems & bundler

### DISCUSSION THREAD

**Discussion by Rendyrenaldi1933** [ruby/rubygems#6646] [Q&A]
~/metasploit-termux $ msfconsole                  Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.                                Could not find nokogiri-1.8.0 in any of the sources                                                 Run `bundle install` to install missing gems.???

**Comment by duckinator** [comment]
Given that the error is generated by running `msfconsole`, this appears to be a metasploit problem.
